README.md

```diff
@@ -84,16 +84,16 @@ When you are architecting an app, try to keep these **key principles** in mind:
 - How can you make it **easy to change**?
 - Can you **anticipate changes** that are likely to occur?
 
->"If you don't develop an architecture, you will get one anyway - and you might not like what you get!"
->-- Northrop, [The Importance of Software Architecture](http://csse.usc.edu/GSAW/gsaw2003/s13/northrop.pdf)
+> "If you don't develop an architecture, you will get one anyway - and you might not like what you get!"
+> -- Northrop, [The Importance of Software Architecture](http://csse.usc.edu/GSAW/gsaw2003/s13/northrop.pdf)
 
 ## Exercise 2: Restructuring A File System
 
 Your file structure is an architectural structure in its own right. Again, it is an _attempt to manage complexity_.
 
 Keeping the Key Principles above in mind, how would you reorganise the files of this repository? (They are intentionally empty files, by the way.)
 
-_**Tip**: if you haven't done so already, you can run `npm install file-icons` so that all your atom icons have icons displaying in the file tree, making them easier to distinguish_
+\_**Tip**: if you haven't done so already, you can run `websemantics/file-icons-js` so that all your files have icons that display in the file tree, making them easier to distinguish. Atom and VS Code also have extensions that do this e.g. Material Icon Theme.
```

Comment:
I think there's a superfluous backslash at the start of this line (line 96), and a missing underscore at the end :)